# Generated by Django 4.2.5 on 2023-09-23 05:39

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('main', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='FAQ',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('question', models.CharField(help_text='Enter question', max_length=150)),
                ('answer', models.TextField(help_text='Enter answer')),
                ('date_added', models.DateTimeField(auto_now_add=True)),
            ],
        ),
        migrations.CreateModel(
            name='News',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('title', models.CharField(help_text='Enter title', max_length=50)),
                ('essence', models.CharField(help_text='Enter essence', max_length=50)),
                ('photo', models.ImageField(blank=True, upload_to='images')),
                ('description', models.TextField(help_text='Enter description')),
            ],
        ),
        migrations.CreateModel(
            name='Review',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('text', models.TextField(help_text='Enter review')),
                ('rating', models.IntegerField(help_text='Enter rating from 0 to 10', validators=[django.core.validators.MinValueValidator(0, message='The number cannot be less than 0'), django.core.validators.MaxValueValidator(10, message='The number cannot be more than 10')])),
                ('date_added', models.DateTimeField(auto_now_add=True)),
                ('author', models.CharField(default='', max_length=20)),
            ],
        ),
    ]
